packages/smart-account/src/signers/Signer.ts

```diff
@@ -11,7 +11,7 @@ import { BytesLike } from '@ethersproject/bytes'
 import { Deferrable } from 'ethers/lib/utils'
 import { TransactionRequest } from '@ethersproject/providers'
 
-export abstract class Signer extends AbstractSigner {
+export abstract class Signer extends EthersSigner {
```

Comment:
Are we using this Signer class directly? 
Why are we extending from Signer and not AbstractSigner?

Reply:
Updated!